DEVELOPERS.rst

```diff
@@ -0,0 +1,68 @@
+For developers of ZODB
```

Comment:
This is no longer published anywhere on the web. Perhaps it could be referenced from a file included in doc/index.rst (.. include: ../DEVELOPERS.rst), and then it might be a good idea to add a link from the PyPI page to this info.

I'd also suggest adding an explicit link to the github repo *somewhere*, probably in this document. If I need to fix a bug (open a PR) in an unfamiliar package, I need to find its repo. First I check the PyPI page, it's often listed as the URL. But if not, I look for a "developing" page in the documentation which should list the repo. Bonus points if that just takes me one click from the PyPI page.